test/fixtures/factories/commit-comment-thread-results.js

```diff
@@ -6,6 +6,7 @@ export function createCommitComment(opts = {}) {
   const o = {
     id: idGen.generate('comment-comment'),
     commitOid: '1234abcd',
+    includeAuthor: true,
```

Comment:
I've been trying to phase out these `test/fixtures/factories` methods in favor of GraphQL builders - the GraphQL builders are easier to write against (the builder structure maps directly to the query!) and they keep up with changes to our queries and the schema. It's tedious as hell, though, so I don't blame you for patching this in 😛